aeon/dj_pipeline/subject.py

```diff
@@ -142,6 +162,14 @@ class SubjectComment(dj.Imported):
     """
 
 
+@schema
+class ExperimentSubject(dj.Manual):
+    definition = """
+    -> Subject
+    experiment_name: varchar(32)  # e.g. social-AEON3
```

Comment:
would having this in a separate table make it less error-prone?

Reply:
Not clear what you mean, what would be your suggested alternative?

Reply:
I'm thinking about a senario of having a upstream lookup table named e.g., `Experiment` where a user can manually enter an experiment name when it's not present, and this table makes a foreign key reference to that one so that a user can just select an experiment via dropdown in SciViz, if that makes sense.